README.md

```diff
@@ -1,9 +1,64 @@
 # Overview
 
-| Docker        | Image                           | Ports                             |
-| :------------ | :------------------------------ | :-------------------------------- |
-| Next.js       |  edu-plattform-mvp_frontend     | 0.0.0.0:25000->3000/tcp           |
-| Hasura        |  hasura/graphql-engine:v1.3.3   | 0.0.0.0:8080->8080/tcp            |
-| Keycloak      |  jboss/keycloak                 | 8443/tcp, 0.0.0.0:28080->8080/tcp |
-| Posgres DB    |  postgres:12                    | 5432/tcp                          |
+| Docker     | Image                        | Ports                             |
+| :--------- | :--------------------------- | :-------------------------------- |
+| Next.js    | edu-plattform-mvp_frontend   | 0.0.0.0:25000->3000/tcp           |
+| Hasura     | hasura/graphql-engine:v1.3.3 | 0.0.0.0:8080->8080/tcp            |
+| Keycloak   | jboss/keycloak               | 8443/tcp, 0.0.0.0:28080->8080/tcp |
+| Posgres DB | postgres:12                  | 5432/tcp                          |
 
+# Development
+
+## Backend
+
+You need a Hasura executable in your PATH. You can use `npm install --global hasura-cli`.
+More information on installing Hasura CLI: https://hasura.io/docs/latest/graphql/core/hasura-cli/install-hasura-cli.html
+
+Start Hasura Console by running:
+
+```
+cd backend
+hasura console --admin-secret myadminsecretkey
+```
+
+And visit http://localhost:9695
+
+### Apply migrations, seeds and metadata
+
+Go and get `exportedseed.sql` (ask somebody) and copy it to `./backend/seeds/default`.
```

Comment:
@ckru-cbb what is the best way to get the current version(s) of this file to every developer machine?